cgmanifest.json

```diff
@@ -10512,7 +10512,7 @@
         "other": {
           "name": "libsmi",
           "version": "0.4.8",
-          "downloadUrl": "ftp://ftp.ibr.cs.tu-bs.de/pub/local/libsmi/libsmi-0.4.8.tar.gz"
+          "downloadUrl": "https://cblmarinerstorage.blob.core.windows.net/sources/core/libsmi-0.4.8.tar.gz"
```

Comment:
According to [this](https://www.ibr.cs.tu-bs.de/projects/libsmi/download.html?lang=de), they moved the package hosting from their website to [a Gitlab repository](https://gitlab.ibr.cs.tu-bs.de/nm/libsmi).

Reply:
yep but there are no releases or tags or even any other branches.

Reply:
Correct. I couldn't find a way to get a stable link to a source tarball, though. I could only get something that's always the archived `master` branch, so it will move with any change to that repo. Having said that, if you know how to get a link for a tarball for a specific commit hash, then that would solve our problem. I couldn't find it, but since it's doable in GitHub, I wouldn't be surprised if GitLab had an equivalent.

Reply:
@PawelWMS Seems like this is the format https://gitlab.ibr.cs.tu-bs.de/nm/libsmi/-/archive/<commit hash>/libsmi-<commit hash>.tar.gz, and I'm not sure why they only have the 'master' branch (I think they've abandoned it? No pushes in 4+ years except for moving the code to Gitlab)